runtime-decompiler/src/main/java/org/jrd/backend/data/Main.java

```diff
@@ -36,71 +39,174 @@ public static void main(String[] args){
                     break;
                 }
             }
-           MainFrameView mainView = new MainFrameView();
+            MainFrameView mainView = new MainFrameView();
             VmDecompilerInformationController controller = new VmDecompilerInformationController(mainView, manager);
         } else {
-            String LISTVMS="-listvms";
-            String LISTCLASSES="-listclasses";
-            for(int i= 0 ; i < args.length; i++){
+            String LISTJVMS = "-listjvms";
+            String LISTPLUGINS = "-listplugins";
+            String LISTCLASSES = "-listclasses";
+            String BASE64 = "-base64bytes";
+            String BYTES = "-bytes";
+            String DECOMPILE = "-decompile";
+            for (int i = 0; i < args.length; i++) {
```

Comment:
What? You loop over args but then break out of the for-loop when you find a parameter.

Reply:
You noted!-)))

Yes....  I initially wrote that cycle as I was thinking that it will process more parameters like --listjvms --decompile blah.ble firstVmFoundInFirstSwithc --listjvms

But then abandoned this idea. Yo are right, current cycle do not have sense. And likely is not going to have, unless multiple switches will be implemented, and then better parser will need to be done/used.

Do you wont me to change it?

I plan to add -help and support javap, and maybe a bit of re-factoring  in this CLI mode.

Thanx!